config/scout.php

```diff
@@ -28,7 +28,7 @@
     |
     */
 
-    'prefix' => env('SCOUT_PREFIX', ''),
+    'prefix' => env('SCOUT_PREFIX', config('app.env').'_'),
```

Comment:
Would prefixing with the application name be helpful? Thinking about once we get Contentful stuff in here too...

Reply:
I'm not 100% sure on the correct answer here, and we'll likely find the best approach as we proceed further. I suspect we'd want different platforms sharing indexes for example if we end up deciding we _do_ want Contentful campaign information integrated with Rogue campaign info. Otherwise when searching for a campaign we'd have to hit 2 different indexes which could get messy. 🤷‍♂️

Reply:
Ohhhhhh! You're totally right! Never mind me. 🙈